adapters/adkernelAdn/adkernelAdn.go

```diff
@@ -108,12 +108,15 @@ func dispatchImpressions(imps []openrtb.Imp, impsExt []openrtb_ext.ExtImpAdkerne
 func compatImpression(imp *openrtb.Imp) error {
 	imp.Ext = nil //do not forward ext to adkernel platform
 	if imp.Banner != nil {
-		return compatBanerImpression(imp.Banner)
+		return compatBanerImpression(imp)
 	}
 	return nil
 }
 
-func compatBanerImpression(banner *openrtb.Banner) error {
+func compatBanerImpression(imp *openrtb.Imp) error {
```

Comment:
Looks like this function name has a typo. I believe it should be `compatBannerImpression` instead of `compatBanerImpression`. Can we also refactor this as part of this PR if it doesn't require a lot of effort?